File: src - c++/Esercizi/EliminaUltimi/EliminaUltimi.cpp
unsigned int Lista::EliminaUltimi(unsigned int n) {
  if (n >= nelem) { //se n >= nelem la lista va svuotata
    unsigned int n = nelem;
    Svuota();
    return n;
  } else //altrimenti implement. questo metodo nei termini di LasciaPrimi()
      return LasciaPrimi(nelem - n);
}

File: src - java/Esercizi/EliminaUltimi/EliminaUltimi.java
	public int eliminaUltimi(int n) {
		if (n >= numEl) { // se n >= numEl la lista va
			              // svuotata
			int nel = numEl;
			svuota();
			return nel;
		} else { // altrimenti implemento questo metodo nei
			     // termini di lasciaPrimi().
			return lasciaPrimi(numEl - n);
		}
	}